### PR TITLE
Rajbir kaur

### PR DESCRIPTION
**Pull request by Rajbir786**

### CODE REVIEW

File: src/main/java/com/naveenautomation/Listeners/WebdriverEvents.java [hunk 1]
```diff
@@ -11,7 +11,7 @@
 import com.naveenautomation.Utils.Log;
 
 public class WebdriverEvents extends TestBase implements WebDriverEventListener {
-
+//munni badnam
```
Comment (PivotITAcademy) [inline src/main/java/com/naveenautomation/Listeners/WebdriverEvents.java:14]: remove this

File: src/main/java/com/naveenautomation/Listeners/WebdriverEvents.java [hunk 2]
```diff
@@ -36,7 +36,7 @@ public void beforeAlertDismiss(WebDriver driver) {
 		// TODO Auto-generated method stub
 		
 	}
-
+//sheela jawan
```
Comment (PivotITAcademy) [inline src/main/java/com/naveenautomation/Listeners/WebdriverEvents.java:39]: remove this

File: src/main/java/com/naveenautomation/Pages/PurchaseGiftCertificatePage.java [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+			return getDomain() + PAGE_URL;
+		}
+
+		@Override
+		protected void isLoaded() {
+			// TODO Auto-generated method stub
+			
+		}
+		public String getTitle() {
+			// TODO Auto-generated method stub
```
Comment (PivotITAcademy) [inline src/main/java/com/naveenautomation/Pages/PurchaseGiftCertificatePage.java:94]: remove this

File: src/main/java/com/naveenautomation/Pages/PurchaseGiftCertificatePage.java [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+
+		
+		@Override
+		protected String getPageUrl() {
+			return getDomain() + PAGE_URL;
+		}
+
+		@Override
+		protected void isLoaded() {
+			// TODO Auto-generated method stub
```
Comment (PivotITAcademy) [inline src/main/java/com/naveenautomation/Pages/PurchaseGiftCertificatePage.java:90]: remove this

File: src/test/java/com/naveenautomation/tests/MyAccountTest.java [hunk 5]
```diff
@@ -39,6 +42,6 @@ public void verifyAlertBannerForInvalidLoginCredentials() {
 
 	@AfterMethod
 	public void teardown() {
-		quitBrowser();
+		//quitBrowser();
```
Comment (PivotITAcademy) [inline src/test/java/com/naveenautomation/tests/MyAccountTest.java:45]: why u r commenting this?

**Review by PivotITAcademy — CHANGES_REQUESTED** [review]
Please make the required changes

File: src/main/java/com/naveenautomation/Listeners/WebdriverEvents.java (hunk 1 above)
Comment (Rajbir786) [reply src/main/java/com/naveenautomation/Listeners/WebdriverEvents.java]: Removed

File: src/main/java/com/naveenautomation/Listeners/WebdriverEvents.java (hunk 2 above)
Comment (Rajbir786) [reply src/main/java/com/naveenautomation/Listeners/WebdriverEvents.java]: Removed

File: src/main/java/com/naveenautomation/Pages/PurchaseGiftCertificatePage.java (hunk 3 above)
Comment (Rajbir786) [reply src/main/java/com/naveenautomation/Pages/PurchaseGiftCertificatePage.java]: Removed

File: src/main/java/com/naveenautomation/Pages/PurchaseGiftCertificatePage.java (hunk 4 above)
Comment (Rajbir786) [reply src/main/java/com/naveenautomation/Pages/PurchaseGiftCertificatePage.java]: Removed

File: src/test/java/com/naveenautomation/tests/MyAccountTest.java (hunk 5 above)
Comment (Rajbir786) [reply src/test/java/com/naveenautomation/tests/MyAccountTest.java]: I have removed comment

**Review by PivotITAcademy — APPROVED** [review]
LGTM

**Pull request merged**